core/src/main/scala/org/broadinstitute/dsde/rawls/googleProject/GoogleProjectRegistrationService.scala

```diff
@@ -109,4 +112,44 @@ class GoogleProjectRegistrationService(protected val ctx: RawlsRequestContext,
         case None => Future.successful(None)
       }
     } yield finalResult
+
+  def getGoogleProjects(billingProjectName: Option[RawlsBillingProjectName]): Future[Seq[GoogleProjectRegistration]] = {
+    val accessibleGoogleProjectsFuture = samDAO
+      .listResourcesWithActions(SamResourceTypeNames.googleProject, SamGoogleProjectActions.readPolicies, ctx)
```

Comment:
why check the `read_policies` action? I think this should just check a simple `read` action. Unfortunately I don't see that already in Sam; you would have to add it.